bitcoinspv/clients/sui/utils.go

```diff
@@ -46,3 +47,48 @@ func BlockHeaderToHex(header wire.BlockHeader) (string, error) {
 func BlockHashToHex(hash chainhash.Hash) string {
 	return "0x" + hex.EncodeToString(hash.CloneBytes())
 }
+
+type bcsEncode []byte
```

Comment:
put types definition at the top of the file (before functions)

Reply:
Done!